app/src/main/res/layout/fragment_user_info_dialog.xml

```diff
@@ -3,7 +3,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    tools:context="com.rage.clamber.Fragments.UserInfoDialogFragment">
+    tools:context=".Fragments.NewUserDialogFragment">
```

Comment:
is this intended to be `.Fragments.NewUserDialogFragment`? Do you not need the whole package path?